src/h3lib/lib/localij.c

```diff
@@ -548,24 +548,24 @@ H3Error H3_EXPORT(experimentalH3ToLocalIj)(H3Index origin, H3Index h3,
  * Failure may occur if the index is too far away from the origin
  * or if the index is on the other side of a pentagon.
  *
- * This function is experimental, and its output is not guaranteed
+ * This function's output is not guaranteed
  * to be compatible across different versions of H3.
  *
  * @param origin An anchoring index for the ij coordinate system.
  * @param out ij coordinates to index.
+ * @param mode Mode, must be 0
  * @param index Index will be placed here on success.
  * @return 0 on success, or another value on failure.
  */
-H3Error H3_EXPORT(experimentalLocalIjToH3)(H3Index origin, const CoordIJ *ij,
-                                           H3Index *out) {
-    // This function is currently experimental. Once ready to be part of the
-    // non-experimental API, this function (with the experimental prefix) will
-    // be marked as deprecated and to be removed in the next major version. It
-    // will be replaced with a non-prefixed function name.
+H3Error H3_EXPORT(localIjToCell)(H3Index origin, const CoordIJ *ij,
+                                 uint32_t mode, H3Index *out) {
+    if (mode != 0) {
+        return E_FAILED;
```

Comment:
Suggestion: make a special `E_INVALID_MODE`? I imagine we'll need this for other cases too, and for `polygonToCells`